coordinators/build.gradle

```diff
@@ -17,6 +17,7 @@ android {
 
 dependencies {
   compile 'com.android.support:support-annotations:23.3.0'
+  compile 'com.android.support:support-v4:23.3.0'
```

Comment:
Speaking as someone who couldn't use this library if it required API 19, I support this dependency. Unless you're talking about inlining the implementation in ViewCompat? I'd be in favor of that.